core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -216,7 +217,6 @@ func (db *DbEthTx) FromTx(tx *Tx) {
 	db.Meta = tx.Meta
 	db.Subject = tx.Subject
 	db.PipelineTaskRunID = tx.PipelineTaskRunID
-	db.MinConfirmations = tx.MinConfirmations
```

Comment:
I think we either need to drop these fields if they are unused, or continue to set them from old data if they are still used. Having them around but unset will just be misleading.